src/services/FileSaveService.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @module services/FileSaveService
+ */
+
+/**
+ * Service for saving files
+ * @class
+ * @author thiloSchlemmer
+ */
+export class FileSaveService {
+	/**
+	 * saves the specified content and makes it for the user available as downloaded file
```

Comment:
[S]aves the specified content and makes it for the user available as a downloaded file